src/main/java/com/example/adventurebackend/controller/ActivityController.java

```diff
@@ -34,7 +34,8 @@ public Activity postActivity(@RequestBody Activity activity) {
 
   @PutMapping("/update/activity/{id}")
   public ResponseEntity<Activity> updateActivity(
-      @PathVariable("id") Long id, @RequestBody Activity activity) {
+      //      @PathVariable("id") Long id, @RequestBody Activity activity) {
+      @PathVariable("id") int id, @RequestBody Activity activity) {
```

Comment:
```suggestion
      @PathVariable int id, @RequestBody Activity activity) {
```

Behøves ikke, virker fint uden :-P